docs/source/importing.rst

```diff
@@ -0,0 +1,62 @@
+Importing Calendars
+===================
+
+Xandikos uses `git` to manage your underlying calendars. This guide shows how to import pre-existing events, such as from popular calendar services as Google Calendar.

```

Comment:
For importing from another service, it's usually better to use a tool like evolution, thunderbird or vdirsyncer than to copy manually to the git repository.

This means that you actually get validation of the ics files.